boards/nxp/s32z2xxdc2/s32z2xxdc2_s32z270.dtsi

```diff
@@ -58,3 +58,55 @@
 &sar_adc1 {
 	vref-mv = <1800>;
 };
+
+&qspi0 {
+	pinctrl-0 = <&qspi0_default>;
+	pinctrl-names = "default";
+	data-rate = "DDR";
+	column-space = <3>;
+	word-addressable;
+	hold-time-2x;
+	a-dll-freq-enable;
+	a-dll-ref-counter = <2>;
+	a-dll-resolution = <2>;
+	a-dll-coarse-delay = <2>;
+	a-dll-fine-delay = <2>;
+	a-dll-mode = "AUTO UPDATE";
+	a-rx-clock-source = "EXTERNAL DQS";
+	ahb-buffers-masters = <0 1 2 3>;
+	ahb-buffers-sizes = <256 256 256 256>;
+	ahb-buffers-all-masters;
+	status = "okay";
+
+	sfp_mdad_0: sfp_mdad_0 {
+		compatible = "nxp,s32-qspi-sfp-mdad";
+		domain-id = <0>;
+	};
+
+	sfp_frad_0: sfp_frad_0 {
+		compatible = "nxp,s32-qspi-sfp-frad";
+		start-address = <0x0>;
+		end-address = <DT_SIZE_M(512)>;
+	};
+
+	s26hs512t: s26hs512t@0 {
```

Comment:
According to the board schematics, rev D has a S76HS512TC0BHB010 and rev B a S71KS512C0B connected to QSPI0 A1 port, and both are hyperram and not hyperflash. Please clarify

Reply:
Maybe there is the difference between the schematic board versions.
The board that I'm using, mounted a hyperflash S26HS512 on board.
I will investigate more about this.
About 'S76HS512TC0BHB010' and 'S71KS512C0B' they are multi-chip package, include both hyperflash chip and hyperram chip. So I believe that the support can work on them although I not have device to test with them.

Reply:
As my investigation, the original schematic design uses S71KS on S32Z27 DC2 but this part could not be sourced during the pandemic so it is replaced by S26HS on board. So there is newer schematic updated to use S26HS.